backend/fms_core/import_tool/_utils.py

```diff
@@ -0,0 +1,8 @@
+from pandas import pandas as pd
+
+def data_row_ids_range(starting_row, df):
```

Comment:
This is too low level to my taste. You should be able to fetch the needed column using the structure and methods provided by panda.

Reply:
it is not fetching the columns.. its just saying that this section is from Row X to Row Y and generating a Range() for it, so we can iterate through it. It's an helper method i create so we can calculate  the rows from the initial header to the end of the dataframe according to its length.